packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -108,13 +108,17 @@ function persistStagesJoinTables({ strapi }) {
   };
 }
 
+const registerWebhookEvents = async ({ strapi }) =>
+  strapi.webhookStore.allowedEvents.set('WORKFLOW_STAGE_CHANGE', 'workflow.stage.change');
+
 module.exports = ({ strapi }) => {
   const workflowsService = getService('workflows', { strapi });
   const stagesService = getService('stages', { strapi });
 
   return {
     async bootstrap() {
       await initDefaultWorkflow({ workflowsService, stagesService, strapi });
+      await registerWebhookEvents({ strapi });
```

Comment:
wonder if we need webhooks to exist during the `initDefaultWorkflow`, maybe we could register the webhooks before, or in the register file (dont know if the webhookStore is initialized by then).

Reply:
it looks like `strapi.webhookStore.allowedEvents` is not available in the register phase but yes I can move it before `initDefaultWorkflow` in bootstrap

Reply:
I would say consistency and we could leave the yup validation in the controllers, and do a separate validation in webhook-store without up. Just say allowedEvents.values().includes(event) would be enough.

Reply:
what do you think? Also, @nathan-pichon

Reply:
I agree, the shape validation is in the controller because it's the entry point of our system. We can do the business validation in services without yup (or having another yup schema dedicated), but here it doesn't seem useful to have a big yup schema to check the events names 🤔

Reply:
And it looks ok to me to add new events in the bootstrap, register phase is the init of the whole system and I see the bootstrap as the setup of this initialized system.